src/sentry/integrations/msteams/card_builder/issues.py

```diff
@@ -53,7 +53,11 @@
 
 class MSTeamsIssueMessageBuilder(MSTeamsMessageBuilder):
     def __init__(
-        self, group: Group, event: Event, rules: Sequence[Rule], integration: RpcIntegration
+        self,
+        group: Group,
+        event: Event | GroupEvent,
```

Comment:
Any chance this could be `BaseEvent` instead?

Reply:
Yep, good call. https://github.com/getsentry/sentry/pull/77586/commits/81b5cc16de3ff2f0df8ecdc8c7043368c2f14c30

Reply:
Looks like this broke typing 🙈 Weird that both subclasses have a `group` attribute, but the base class doesn't.